src/pocketmine/utils/Terminal.php

```diff
@@ -129,6 +131,7 @@ protected static function getFallbackEscapeCodes(){
 		self::$COLOR_LIGHT_PURPLE = "\x1b[38;5;207m";
 		self::$COLOR_YELLOW = "\x1b[38;5;227m";
 		self::$COLOR_WHITE = "\x1b[38;5;231m";
+		self::$COLOR_DARK_YELLOW = "\x1b[38;5;226m";
```

Comment:
I'm a little unclear why different codes are used for `tput` escape codes and ANSI escape codes. All the others match except this one, but here we have `226` vs `178`.

Reply:
I was not aware that the two were linked in that way, I'll update it now.

Reply:
well both are doing the rounds with 256color so they should be the same.

Reply:
I fixed it in the recent commit